software/UMS/index.rst

```diff
@@ -24,30 +24,62 @@ Select "User Managed Software Program" from the drop down menu to begin.
+  |            | Ascent/Conduit |                                            |                                                                                |
+  |            | Adios2 dev     |                                            |                                                                                |
+  +------------+----------------+--------------------------------------------+--------------------------------------------------------------------------------+
+  | gen119     | NVIDIA RAPIDS  | Benjamin Hernandez (hernandezarb@ornl.gov) | https://developer.nvidia.com/rapids                                            |
+  |            | BlazingSQL     |                                            | https://blazingsql.com/                                                        |
+  +------------+----------------+--------------------------------------------+--------------------------------------------------------------------------------+
+  | ums012     | SOLLVE         | Felipe Jaramillo (cabarcas@udel.edu)       |                                                                                |
+  |            |                |                                            |                                                                                |
+  |            | LLVM           | Sunita Chandrasekaran (schandra@udel.edu)  |                                                                                |
+  +------------+----------------+--------------------------------------------+--------------------------------------------------------------------------------+
+  | ums013     | Julia          | Valentin Churavy  (vchuravy@mit.edu)       | https://julialang.org                                                          |
+  +------------+----------------+--------------------------------------------+--------------------------------------------------------------------------------+
+  | ums014     | UPC++          | Paul H. Hargrove (phhargrove@lbl.gov)      | Issue Tracker:     https://upcxx.lbl.gov/issues                                |
+  |            |                |                                            |                                                                                |
+  |            |                |                                            | Support Forum:     https://groups.google.com/g/upcxx                           |
+  +------------+----------------+--------------------------------------------+--------------------------------------------------------------------------------+
+  | ums015     | DPC++ for HIP  | Gordon Brown (gordon@codeplay.com)         |                                                                                |
+  +------------+----------------+--------------------------------------------+--------------------------------------------------------------------------------+
+  
+.. tabbed:: Spock
```

Comment:
Since Spock is decommissioned, should we remove the tab from this sheet?

Reply:
Removed Spock references